### PR TITLE
Add smoke tests

### PR DESCRIPTION
**Pull request by pamelafox**

### CODE REVIEW

File: {{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml [hunk 1]
```diff
@@ -67,3 +69,39 @@ jobs:
...
+    smoketests:
+      runs-on: ubuntu-latest
+      needs: build
+      steps:
+
+        - name: Basic smoke test (curl)
+          env:
+            URI: ${{needs.build.outputs.uri}}
+          run: |
+            echo "Sleeping 1 minute due to https://github.com/Azure/azure-dev/issues/2669"
```
Comment (pamelafox) [inline {{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml:89]: I do wonder if we want to do this a fancier way. We could even write a Python script using tenacity to check for 200 status with backoff and 10 attempts, that sort of thing. Or we could do that in bash (barf).

**Comment by pamelafox** [comment]
Tested this update here: https://github.com/Azure-Samples/azure-flask-postgres-flexible-appservice/pull/13/files

File: {{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml (hunk 1 above)
Comment (kjaymiller) [reply {{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml:89]: I think we can make it cooler later... this works for now (unless it doesn't)

Comment (pamelafox) [reply {{cookiecutter.__src_folder_name}}/.github/workflows/azure-dev.yml:89]: It does seem to work so far! We'll see what happens when we cruft update on all 17 repos.

**Review by kjaymiller — APPROVED** [review]
Looks great! Happy to get smoke-tests added to our demos!

**Pull request merged**